### ISSUE TITLE
Missing tombstones prevent game launch

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#3]
```
(clone of 
http://fireforge.net//tracker/?func=detail&atid=125&aid=274&group_id=11 )

Submitted by:
Christopher Vagnetoft (noccy80)

"Detailed description
Several times tonite a multiplayer game has failed to load since tombstones 
haven't been found on one or more of the
players' computers. A missing tombstone should not cause the load to fail, but 
rather revert to a default."
```

-----
Original issue reported on code.google.com by `sheepyluva` on 28 Jun 2010 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.9.14
```

-----
Original comment by `mario.li...@gmail.com` on 30 Jul 2010 at 4:24
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified.
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 7:50
* Changed state: **Verified**
